### ISSUE TITLE
Preview for images without version

### ISSUE THREAD

**Issue by 00dav00** [zooppa/administrate-field-carrierwave#10]
Correct me if I'm wrong, but it seems like images that don't have ata least one version won't be able to have a preview because [show_preview?](https://github.com/zooppa/administrate-field-carrierwave/blob/master/lib/administrate/field/carrierwave.rb#L38) method checks that a version exists.

```ruby
      def show_preview?
        data.model.persisted? && file.version_exists?(image) && data.file.present?
      end
```

I think it would be great if this is only checked in case `image` is option is present, otherwise validating that the model is persisted and the file exists should be enough. WDYT?

**Comment by michelegera** [comment]
@00dav00 I just pushed a change that should address this.

Could you check out the `master` branch locally and see if this fixes your issue?

You’d have to clone this repo on your computer and change the `Gemfile` of the application that uses the plugin:

```
gem 'administrate-field-carrierwave', path: '../path/to/cloned/repo'
```

Then `bundle` and restart your app.

**Comment by 00dav00** [comment]
I used the master branch and it worked like a charm, Thanks a lot @michelegera!

```ruby
gem 'administrate-field-carrierwave', git: 'git@github.com:zooppa/administrate-field-carrierwave.git', branch: 'master'
```

![file](https://user-images.githubusercontent.com/9206778/47332978-db4b4c80-d646-11e8-8469-289b011d4e3e.png)

**Comment by michelegera** [comment]
@00dav00 I’ve just released [v0.3.4](https://rubygems.org/gems/administrate-field-carrierwave/versions/0.3.4) of the gem which incorporates this change.

You can upgrade and go back to `gem 'administrate-field-carrierwave'` in your `Gemfile` 😄
